test/language/default-parameters/undefined-parameters-doesnt-replace-default.js

```diff
@@ -0,0 +1,17 @@
+// Copyright (C) 2015 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 13.3.3.7
+description: >
+  Undefined parameters doesn't replace default values.
```

Comment:
I think this could use more formal justification

    > 24. If hasDuplicates is true, then
    >     [...]
    > 25. Else,
    >     a. Let formalStatus be IteratorBindingInitialization for formals with
    >        iteratorRecord and env as arguments.

    > ES6 13.3.3.6 Runtime Semantics: IteratorBindingInitialization
    >
    > SingleNameBinding : BindingIdentifier Initializeropt
    >
    > [...]
    > 6. If Initializer is present and v is undefined, then
    >    [...]